sdk/eventhub/azure-eventhubs/tests/livetest/synctests/test_consumer_client.py

```diff
@@ -0,0 +1,80 @@
+import time
+import pytest
+import threading
+from azure.eventhub import EventData
+from azure.eventhub import EventHubConsumerClient
+from azure.eventhub._eventprocessor.local_partition_manager import InMemoryPartitionManager
+
+
+@pytest.mark.liveTest
```

Comment:
There is an official marker "live_test_only" that auto-skip the test if the env variable `AZURE_TEST_RUN_LIVE` is not set. Maybe worth being consistent with other packages?

Reply:
Not blocking for preview5, solving